game-core/src/test/java/games/strategy/engine/framework/startup/ui/PlayerTypeTest.java

```diff
@@ -6,15 +6,18 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import games.strategy.engine.player.Player;
+import games.strategy.triplea.settings.ClientSetting;
 import java.util.List;
 import org.hamcrest.Matchers;
 import org.hamcrest.core.IsCollectionContaining;
 import org.junit.jupiter.api.Test;
+import org.sonatype.goodies.prefs.memory.MemoryPreferences;
 
 class PlayerTypeTest {
 
   @Test
   void playerTypes() {
+    ClientSetting.setPreferences(new MemoryPreferences());
```

Comment:
There is an abstract test that is typically used when the code under test depends on ClientSetting. Did you intentionally skip using that in favor of this setter, if so I'm curious of the benefit of doing a client setting test this way vs extending the base test class.

Reply:
I was just trying to get this test to pass.  I don't know about the abstract test you are talking about nor do I know if this test will work using that abstract test.

Reply:
See `AbstractClientSettingTestCase`